engine/src/main/java/io/zeebe/engine/processing/deployment/model/validation/ExpectedValidationResult.java

```diff
@@ -56,7 +56,7 @@ public boolean matches(final ValidationResult result) {
       match = false;
     }
 
-    if (!expectedMessage.equals(result.getMessage())) {
+    if (!result.getMessage().startsWith(expectedMessage)) {
```

Comment:
why is this okay?

Reply:
Good catch :eyes: The class should be used in tests only. I'll move it. 

I changed the verification to simplify the tests (i.e. don't verify the whole message) and to make it more stable for updates (i.e. if the error message changes in the FEEL engine).